release-notes.md

```diff
@@ -117,6 +117,14 @@ This document contains release notes per zib, indicating differences with their
 ## zib-TimeInterval
 * In the previous release, the concepts of this partial zib were mapped directly in the profiles where they are used. In this release the usage of this partial zib has been expanded and profiles have been created to aid the usage in profiles in the various situations that might occur.
 
+
+## zib-TreatmentObjective
+* Removed identifier mapping on `Goal.identifier`.
+* DesiredHealthcareResult element moved to `Goal.description.text`.
+* Removed the mapping on `Goal.expressedBy` and the reference to practitionerRole.
+* The reference to FunctionalOrMentalStatus is mapped inline on the goal resource elements (see comment on the root element for the explanation).
+* The medicalDevice extension is mapped on `Goal.target`.
```

Comment:
The medicalDevice extension is part of the new inline mapping of FunctionalOrMentalStatus, correct? Doesn't have to be named here?

Reply:
Fixed